lib/config/presets/internal/default.ts

```diff
@@ -141,11 +141,11 @@ export const presets: Record<string, Preset> = {
     description: 'Disable Renovate Dependency Dashboard creation.',
   },
   disableDevDependencies: {
-    description: 'Do not update `devDependencies` versions/ranges.',
+    description: 'Do not update `devDependencies`/`dev` versions/ranges.',
```

Comment:
```suggestion
    description: 'Do not update development dependencies.',
```

Reply:
Hi! That's correct, but it turns out recent versions of Poetry use `dev` instead. I think this is related to Poetry usage of [dependency groups](https://python-poetry.org/docs/managing-dependencies/#dependency-groups) Do you want me to add both? I can do that!

Reply:
No harm in doing so, as there's no chance it will ever be _wrong_ to include it